pkgs/modules/docker/etc/containerd.toml

```diff
@@ -1,253 +1,9 @@
-disabled_plugins = []
-imports = []
-oom_score = 0
-plugin_dir = ""
-required_plugins = []
-root = ""
-state = "/run/containerd"
-temp = ""
 version = 2
-
-[cgroup]
-  path = ""
-
-[debug]
-  address = ""
-  format = ""
-  gid = 0
-  level = ""
-  uid = 0
+root = "" # Set in the commandline
+state = "/run/containerd"
 
 [grpc]
   address = "/run/containerd/containerd.sock"
   gid = 1000
   max_recv_message_size = 16777216
   max_send_message_size = 16777216
-  tcp_address = ""
-  tcp_tls_ca = ""
-  tcp_tls_cert = ""
-  tcp_tls_key = ""
-  uid = 1000
-
-[metrics]
-  address = ""
-  grpc_histogram = false
-
-[plugins]
-
-  [plugins."io.containerd.gc.v1.scheduler"]
```

Comment:
most of this config got in the way of buildkit, for reasons i don't understand yet.